# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../pdns/pdns-2.9.22-gcc44.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

This is a Gentoo patch from:
http://sources.gentoo.org/cgi-bin/viewvc.cgi/gentoo-x86/net-dns/pdns/files/pdns-2.9.22-gcc44.patch?revision=1.1
diff -ur pdns-2.9.22.orig/pdns/dnsparser.cc pdns-2.9.22/pdns/dnsparser.cc
--- pdns-2.9.22.orig/pdns/dnsparser.cc	2008-11-15 22:32:46.000000000 +0200
+++ pdns-2.9.22/pdns/dnsparser.cc	2009-07-06 22:09:02.000000000 +0300
@@ -16,6 +16,7 @@
     Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 */
 
+#include <stdio.h>
 #include "dnsparser.hh"
 #include "dnswriter.hh"
 #include <boost/lexical_cast.hpp>
diff -ur pdns-2.9.22.orig/pdns/rcpgenerator.cc pdns-2.9.22/pdns/rcpgenerator.cc
--- pdns-2.9.22.orig/pdns/rcpgenerator.cc	2008-11-15 23:20:27.000000000 +0200
+++ pdns-2.9.22/pdns/rcpgenerator.cc	2009-07-06 22:07:26.000000000 +0300
@@ -16,6 +16,7 @@
     Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 */
 
+#include <stdio.h>
 #include "rcpgenerator.hh"
 #include "dnsparser.hh"
 #include "misc.hh"

